### PR TITLE
Tabs block

### PR DESCRIPTION
**Pull request by mind-adobe**
Add tabs block def

Test URLs:
- Before: https://main--aem-block-collection-xwalk--adobe-rnd.hlx.page
- After: https://tabs-block--aem-block-collection-xwalk--adobe-rnd.hlx.page

### CODE REVIEW

File: component-definition.json [hunk 1]
```diff
@@ -167,6 +167,36 @@
...
+          }
+        },
+        {
+          "title": "Tabs Item",
+          "id": "tabs-item",
+          "plugins": {
+            "xwalk": {
+              "page": {
+                "resourceType": "core/franklin/components/block/v1/block/item",
+                "template": {
```
Comment (msagolj) [inline component-definition.json:193]: add a placeholder value for tabtile or block code fails with an error.

**Review by msagolj — APPROVED** [review]

File: component-definition.json [hunk 2]
```diff
@@ -167,6 +167,37 @@
...
+                  "name": "Tab",
+                  "model": "tabs-item",
+                  "tabName" : "Tab Name"
```
Comment (buuhuu) [inline component-definition.json]: better add an empty `<p>&nbsp;</p>` for the content so that the tab content is immediately in context editable.

**Review by buuhuu — COMMENTED** [review]
Not sure how useful Tabs are with only richtext. But at least this is something to get started with. Thank you.

File: component-definition.json (hunk 2 above)
Comment (mind-adobe) [reply component-definition.json]: Added a placeholder

**Review by msagolj — APPROVED** [review]

**Pull request merged**